requirements_dev.txt

```diff
@@ -1,14 +1,17 @@
 pytest>=6.0
+pytest-runner
+pytest-cov
 flake8
+tox
 pytest-flake8
 ipython
 pytest-notebook
 nbsphinx
 nbval>=0.9.6
 nbconvert
-sphinx>=1.7
-bumpversion
-twine
+sphinx>=1.8.5
+bump2version
```

Comment:
`bump-my-version` is the new maintained fork for this project (`bump{2}version` haven't received updates in years!). I only found this out a few weeks ago!